### PR TITLE
openjdk-23, openjfx-23: new, 23+ga

### PR DESCRIPTION
**Pull request by xtexx**
Topic Description
-----------------

- openjfx-23: new, 23+ga
- openjdk-23: new, 23+ga

Package(s) Affected
-------------------

- openjdk-23: 3:23+ga
- openjfx-23: 23+ga

Security Update?
----------------

No

Build Order
-----------

```
#buildit openjdk-23:+stage2 openjdk-23 openjfx-23
```

Test Build(s) Done
------------------

**Primary Architectures**

- [x] AMD64 `amd64`
- [x] AArch64 `arm64`
- [x] LoongArch 64-bit `loongarch64`

**Secondary Architectures**

- [x] Loongson 3 `loongson3`
- [x] PowerPC 64-bit (Little Endian) `ppc64el`
- [x] RISC-V 64-bit `riscv64`

### CODE REVIEW

**Comment by xtexx** [comment]
LGTM on AMD64 by starting Minecraft 1.21 vanilla

**Comment by MingcongBai** [comment]
Tested good with Minecraft on LoongArch.

File: lang-java/openjfx-23/autobuild/gradle.properties [hunk 1]
```diff
@@ -0,0 +1,5 @@
+COMPILE_WEBKIT = true
+COMPILE_MEDIA = true
+BUILD_JAVADOC = true
+BUILD_SRC_ZIP = true
+
```
Comment (MingcongBai) [inline lang-java/openjfx-23/autobuild/gradle.properties]: Extraneous newline.

File: lang-java/openjfx-23/autobuild/defines [hunk 2]
```diff
@@ -0,0 +1,8 @@
+PKGNAME=openjfx-23
+PKGSEC=java
+PKGDEP="openjfx-common ffmpeg gst-plugins-base-1-0 gtk-2 openjdk qt-5 webkit2gtk"
+BUILDDEP="cmake gperf ruby unzip"
+PKGDES="Open source implementation of JavaFX"
```
Comment (MingcongBai) [inline lang-java/openjfx-23/autobuild/defines]: Please add "(version 23)" or something similar.

File: lang-java/openjfx-23/autobuild/defines [hunk 3]
```diff
@@ -0,0 +1,8 @@
+PKGNAME=openjfx-23
+PKGSEC=java
+PKGDEP="openjfx-common ffmpeg gst-plugins-base-1-0 gtk-2 openjdk qt-5 webkit2gtk"
```
Comment (MingcongBai) [inline lang-java/openjfx-23/autobuild/defines]: Use `gstreamer` for all GStreamer dependencies.

File: lang-java/openjdk-23/autobuild/defines.stage2 [hunk 4]
```diff
@@ -0,0 +1,14 @@
+PKGNAME=openjdk-23
+PKGSEC=java
+PKGDEP="ca-certs nss xdg-utils hicolor-icon-theme alsa-lib gtk-2 cups \
+        fontconfig unzip zip cpio giflib"
+BUILDDEP="openjdk-22"
+PKGDES="OpenJDK Java Runtime Environment (JRE), Java Development Environment (JDK), and IcedTea-Web"
```
Comment (MingcongBai) [inline lang-java/openjdk-23/autobuild/defines.stage2]: Please mark version in description - also, are you sure that IcedTea is included?

**Review by MingcongBai — CHANGES_REQUESTED** [review]

**Review by MingcongBai — APPROVED** [review]

**Pull request merged**